.travis.yml

```diff
@@ -32,8 +32,12 @@ env:
     - TOX_ENV=py35-backend_mysql PYTHON_VERSION=3
     - TOX_ENV=py35-backend_sqlite PYTHON_VERSION=3
     - TOX_ENV=py35-backend_postgres PYTHON_VERSION=3
+    - TOX_ENV=py36-backend_mysql PYTHON_VERSION=3
```

Comment:
Also, remember we would be just testing a single python 3 version, which would be either 3.6 and then updated to 3.7 instead of 3.5 . So you can remove py35 lines